websites/P/PaperMC/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "387062216030945281",
+		"name": "Romvnly#6466"
+	},
+	"service": "PaperMC",
+	"description": {
+		"en": "PaperMC improves Minecraft's ecosystem with fast, secure software and an expanding plugin API, providing quick releases and helpful support as the most widely used, performant, and stable software available."
+	},
+	"url": [
+		"papermc.io",
+		"docs.papermc.io",
+		"jd.papermc.io",
+		"hangar.papermc.io",
+		"hangar.papermc.dev",
+		"forums.papermc.io"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/SzuS0AE.png",
```

Comment:
Logos should be [`512x512px`](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo)